settings.gradle.kts

```diff
@@ -3,6 +3,8 @@ pluginManagement {
         mavenCentral()
         gradlePluginPortal()
     }
+
+    includeBuild("build-logic")
```

Comment:
One small nit that I'll throw in the air for @paul-dingemans to decide: we can put this build-logic into any folder. Not sure if the preference is to have all the "modules" visible, or only the main project source code. I usually prefer to keep things separate: i.e. GitHub stuff in `.github`, Gradle stuff in `gradle/*`, Documentation in `docs` etc. With `buildSrc` our hands were tied, there was no choice, Gradle hardcoded that path, but with freedom given I personally prefer to migrate `/buildSrc` to `/gradle/plugins` folder.

Reply:
We put publication & test logic into `build-logic` for almost modules to reuse.

Reply:
Yep, I'm just saying to follow this step:
https://mobile.twitter.com/jeoj/status/1561994066530803712

(`@jeoj` is a master working at Gradle)

Reply:
TIL, thanks for your sharing. I think we can keep the current state to let users easily understand, will migrate to the idiomatic way one day.